controller/src/main/java/com/softserve/borda/config/jwt/JwtProvider.java

```diff
@@ -19,7 +15,7 @@
     private String jwtSecret;
 
     public String generateToken(String login) {
-        Date date = Date.from(LocalDate.now().plusDays(15).atStartOfDay(ZoneId.systemDefault()).toInstant());
+//        Date date = Date.from(LocalDate.now().plusDays(15).atStartOfDay(ZoneId.systemDefault()).toInstant());
```

Comment:
Remove comment

Reply:
It is just that 'date' variable was never used. Now I uncommented it together with line where it should be used.